components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1507,7 +1507,7 @@ void RewardsServiceImpl::SetRewardsMainEnabledPref(bool enabled) {
 
 void RewardsServiceImpl::SetRewardsMainEnabledMigratedPref(bool enabled) {
   profile_->GetPrefs()->SetBoolean(
-      prefs::kBraveRewardsEnabledMigrated, enabled);
+      prefs::kBraveRewardsEnabledMigrated, true);
```

Comment:
Parameter is no longer used

Reply:
this is no longer the case after rebasing